playbooks/logging.yml

```diff
@@ -70,13 +70,12 @@
   tasks:
     - include: ../roles/neutron-common/tasks/logging.yml
 
-# TODO: These roles are missing logging configs!!!
-#- name: horizon logging
-#  hosts: controller
-#  roles:
-#  - role: ../roles/logging-config
-#    service: horizon
-#    logdata: "{{ horizon.logs }}"
+- name: horizon logging
+ hosts: controller
+ roles:
+ - role: ../roles/logging-config
+   service: horizon
+   logdata: "{{ horizon.logs }}"
```

Comment:
Looks like some spacing is weird here. The `h` in `hosts` should line up with the  `n` from `name`.